Casks/f/flexoptix.rb

```diff
@@ -1,15 +1,19 @@
 cask "flexoptix" do
+  arch arm: "arm64", intel: "x64"
+  download_suffix = on_arch_conditional arm: "-arm64", intel: ""
```

Comment:
```suggestion
  arch_suffix = on_arch_conditional arm: "-arm64"
```

We can omit the empty string and rename the variable to `arch_suffix` for consistency.